tools/ci_build/github/azure-pipelines/win-qnn-arm64-ci-pipeline.yml

```diff
@@ -75,12 +75,23 @@ jobs:
     displayName: 'Build'
     inputs:
       scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
-      arguments: '--config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_tests --cmake_generator "Visual Studio 17 2022" --use_qnn --qnn_home $(QnnSDKRootDir) --parallel'
-      workingDirectory: '$(Build.BinariesDirectory)'
+      arguments: >-
+        --config $(BuildConfig)
+        --build_dir $(Build.BinariesDirectory)
+        --cmake_generator "Visual Studio 17 2022"
+        --use_qnn
+        --qnn_home $(QnnSDKRootDir)
+        --update --build --parallel
```

Comment:
adding `--build_java` here failed. it seems like Java is not installed on the build agent.

https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1502577&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=e10d0171-a9eb-5804-8d7d-28924eceb401&l=979

Reply:
If it's important we could update the image to add it, but if the Java binding is tested elsewhere that seems optional unless there's some concern about it not working on arm64.